src/main/java/tech/ypsilon/bbbot/config/CommandsSubconfig.java

```diff
@@ -0,0 +1,28 @@
+package tech.ypsilon.bbbot.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+
+@Data
+@Builder
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor
+public class CommandsSubconfig {
+
+    @JsonProperty("bdaychannel")
+    private Long birthdayChannel;
+
+    @JsonProperty("bday")
+    private BDaySubconfig birthday;
+
+    @Data
+    @Builder
+    @EqualsAndHashCode
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class BDaySubconfig {
```

Comment:
Just for consistency, this class should be moved to a separate file.

Reply:
see c48b130